src/main/java/com/lootfood/entity/City.java

```diff
@@ -0,0 +1,30 @@
+package com.lootfood.entity;
```

Comment:
for entities, repositories, and interfaces that you are using for working with DB, create next package structure and move them all there:
`com.lootfoof.db`
`com.lootfoof.db.entity` - for entities
`com.lootfoof.db.repository` - for repositories
`com.lootfoof.db.repository.custom` - for custom realization of repositories

and don't create separate package for each realization:
`com.lootfood.repository.city`. `com.lootfoof.db.repository` - is enouph